examples/java/build.gradle

```diff
@@ -27,8 +27,14 @@ interface for processing virtually any size data. This
 artifact includes all Apache Beam Java SDK examples."""
 
 /** Define the list of runners which execute a precommit test. */
-// https://issues.apache.org/jira/browse/BEAM-3583
-def preCommitRunners = ["dataflowRunner", "dataflowStreamingRunner", "directRunner", "flinkRunner", "sparkRunner"]
+// apexRunner - https://issues.apache.org/jira/browse/BEAM-3583
+def preCommitRunners = ["directRunner", "flinkRunner", "sparkRunner"]
+// Dataflow runner based tests are defined in their own subprojects so that
+// they can run in parallel because they are slow.
```

Comment:
$0.02: I love having them in separate projects anyhow. No slapping together strings to come up with magic strings.